### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_Resources_Keyboard:The tab focus is not visible after deploy a server button.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#3142]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:-A11y_AzureDataStudioStayHealthy;-A11ySev2;-A11yTCS;-A11yMAS;-Benchmark-DesktopApp;-MAS2.4.7;-SQL Azure Data Studio;-FTP;-Keyboard;-STP;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.4.7

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section "History tab" using tab key,

**Actual:**
The tab focus after deploy a server button is not visible and when screen reader is on the invisible focus make an announcement of a radio button and on the next tab key the focus is on the history tab under Resources section.

**Expected:**
The tab focus after deploy a server button should be on the history tab and the screen reader should make the announcement of it when in focus under Resources section 

**User Impact:**
Keyboard users will not know where the focus has been moved too. 

**Attachment link for Reference:**
